templates/master/01-master-container-runtime/_base/files/crio.yaml

```diff
@@ -348,3 +259,4 @@ contents:
 
     # The port on which the metrics server will listen.
     metrics_port = 9537
+  
```

Comment:
removing a trailing newline?  I'd rather keep it, but I dunno if Go's YAML parser cares or not.